ci/docker/README.md

```diff
@@ -9,30 +9,60 @@ overwrite the one on Docker Hub should they differ.
 
 # Dockerized DAML SDK
 
-Digital Asset's [DAML SDK](https://docs.daml.com/) in a can
+> This image is not supported for production use-cases. Please contact Digital
+> Asset to obtain supported production-ready artifacts.
+
+Digital Asset's [DAML SDK](https://docs.daml.com/) in a can.
 
 ## Tags
 
-* `${SDK_VERSION}` (_Alpine Linux_)
+* `${SDK_VERSION}`
+
+> Starting with tag 1.7.0, these images are signed.
+
+## Verifying signatures
+
+You can configure your Docker client to only pull & run signed Docker images by
+setting the `DOCKER_CONTENT_TRUST` environment variable to 1.
+
+This, however, only checks that the image is signed. If you want to further
+check the provenance of the signature, you can use `docker trust inspect
+--pretty digitalasset/daml-sdk:$TAG`; you should see a signer called
+`automation` with the key
+`533a6e09faa512f974f217668580da1ceb6aa5b00aad34ea1240afc7d249703f` and a
+repository key of
+`f5dc2aee6aed2d05d7eda75db7aa2b3fac7fc67afbb880d03535d5a5295a0d3b`.
 
 ## Quick start
 
 * Ensure Docker is [installed](https://www.docker.com/get-started)
 * Check out existing demo DAML project (or use your own):
-```
-git clone https://github.com/digital-asset/ex-bond-trading.git
-cd ex-bond-trading
-```
+  ```
+  git clone https://github.com/digital-asset/ex-bond-trading.git
+  cd ex-bond-trading
+  ```
 * Run DAML scenarios:
-```
-docker run --rm -it -v $PWD:/data digitalasset/daml-sdk:${SDK_VERSION} bash -c "~/.daml/bin/daml test --files /data/src/main/daml/Test.daml"
-```
+  ```
+  DOCKER_CONTENT_TRUST=1 docker run --rm -it -v $PWD:/data digitalasset/daml-sdk:$SDK_VERSION bash -c "cd \$(mktemp -d) && cp -r /data/* ./ && DAML_SDK_VERSION=$SDK_VERSION daml test"
```

Comment:
The `cp` bit seems to be necessary because we run under a non-root user in the container, and they therefore don't get any write access on the mounted volume (where they would otherwise write the .daml directory).

Reply:
Can we clone in the container to avoid that?

Reply:
That was my first though, but `git` is not installed in the container. Given that we only recommend this for CI use-cases "where image caching makes sense", starting with an `apt install` seems weird. Also the default user is not root so doesn't actually have apt rights. This copy thing seemed like a lesser evil.

And if you're wondering why I'm changing this at all, the existing command in the current description relies on executing a single file directly without a project, which worked fine for scenarios but hits a "Script: not found" error now that we need a project file to specify script as an explicit dependency. Does `daml test` have flags to add dependencies? Would we recommend that over copying? I imagine it wouldn't scale very well with additional dependencies over time, but perhaps daml project dependencies don't change that much?

Reply:
I’m fine with copying, just wanted to understand if there was an easy way to avoid it but messing around with dependencies, `apt` or something else seems worse so let’s stick to that.